config/config.exs

```diff
@@ -26,15 +26,13 @@ config :nerves, source_date_epoch: "1597551838"
 
 config :logger, backends: [RingLogger]
 
-if Mix.target() == :host or Mix.target() == :"" do
+if Mix.target() == :host do
   import_config "host.exs"
 else
   import_config "target.exs"
 end
 
 config :hello_nerves,
-  target: Mix.target(),
-  env: Mix.env(),
```

Comment:
https://github.com/TORIFUKUKaiou/hello_nerves/blob/eca16d8bc3cd37512a5099407ec7b9d8c9c0e8eb/lib/hello_nerves/application.ex#L37

I find that :env is used.
I added with https://github.com/kentaro/mix_tasks_upload_hotswap .
WDYT?